classes/class-wc-connect-shipping-method.php

```diff
@@ -258,6 +263,8 @@ private function add_fallback_rate( $service_settings ) {
 
 		public function calculate_shipping( $package = array() ) {
 
+			$this->debug( 'WooCommerce Services debug mode is on - to hide these messages, turn debug mode off in the settings.' );
```

Comment:
Should we be more clear about what "the settings" is? (This could be a bit misleading since the merchant wouldn't actually find it under WooCommerce -> Settings.) Perhaps even a link?

Reply:
Added a link in https://github.com/Automattic/woocommerce-services/pull/1303/commits/3e12096187780664bd05d195279bd0c7db6e6d88.